bitex/formatters/bitfinex.py

```diff
@@ -29,32 +32,100 @@ def ticker(self):
 
     def order_book(self):
         """Return namedtuple with given data."""
-        raise NotImplementedError
+        data = self.json()
+        asks = []
+        bids = []
+        if 'bids' in data:  # api version 1
+            for i in data['asks']:
+                asks.append([i['price'], i['amount']])
+            for i in data['bids']:
+                bids.append([i['price'], i['amount']])
+        else:   # api version 2
+            for i in data:
+                if i[2][1] == '-':
+                    bids.append([i[0], i[2]])
+                else:
+                    asks.append([i[0], i[2][1:]])
```

Comment:
this is also hard to read and understand what's going on. Also, why would the order books come in two layouts ?

For this particular case, try something like this:
```python
for price, count, size in data:`  # You can unpack lists and tuples that you iterate over directly
    if size.startswith('-'):
        bids.append(price, size)
        continue
    asks.append(price, size)
```

Reply:
api maybe v1 or v2, I just complete v1, and v2 only order_book.
if only v2, your code is better. but your default is v1